GAD-7.json

```diff
@@ -18,25 +25,9 @@
       "code": [
         {
           "code": "69725-0",
-          "display": "Feeling nervous, anxious or on edge",
           "system": "http://loinc.org"
         }
       ],
-      "extension": [
-        {
-          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
-          "valueCodeableConcept": {
-            "coding": [
-              {
-                "system": "http://hl7.org/fhir/questionnaire-item-control",
-                "code": "drop-down",
-                "display": "Drop down"
-              }
-            ],
-            "text": "Drop down"
-          }
-        }
-      ],
```

Comment:
I confirmed that https://github.com/asbi-cds-tools/questionnaire-to-survey doesn't use this - it only reads item.type . fyi @achen2401